BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -23,42 +23,75 @@ public struct ElementContent {
         storage = builder
     }
 
-    // MARK: Measurement & Children
+    fileprivate init(storage: ContentStorage) {
+        self.storage = storage
+    }
+
+    func measure(
+        in constraint: SizeConstraint,
+        with environment: Environment,
+        state: ElementState
+    ) -> CGSize {
+        autoreleasepool {
+            self.storage.measure(in: constraint, with: environment, state: state)
+        }
+    }
+
+    public var childCount: Int {
+        storage.childCount
+    }
+
+    func performLayout(
+        in size: CGSize,
+        with environment: Environment,
+        state: ElementState
+    ) -> [LayoutResultNode] {
+        autoreleasepool {
+            storage.performLayout(
+                in: size,
+                with: environment,
+                state: state
+            )
+        }
+    }
+}
+
+extension ElementContent {
 
     /// Measures the required size of this element's content.
     /// - Parameters:
     ///   - constraint: The size constraint.
     ///   - environment: The environment to measure in.
     /// - returns: The layout size needed by this content.
     public func measure(in constraint: SizeConstraint, environment: Environment) -> CGSize {
-        measure(
-            in: constraint,
-            environment: environment,
-            cache: CacheFactory.makeCache(name: "ElementContent")
-        )
-    }
 
-    func measure(in constraint: SizeConstraint, environment: Environment, cache: CacheTree) -> CGSize {
-        storage.measure(in: constraint, environment: environment, cache: cache)
-    }
+        let element = MeasurementElement(content: self)
```

Comment:
Why is this created outside of the `autoreleasepool`?

Reply:
Probably shouldn't be, moving in